src/login.js

```diff
@@ -76,7 +78,6 @@ export async function login({
     case "google":
     case "linkedin":
     case "okta":
-      return signonWithSso({ provider: method, redirect });
```

Comment:
calling with this signature is the same -- `signonWithSso({ provider: method, redirect, providerId });`